cli/serve/serve.go

```diff
@@ -200,8 +202,29 @@ func serverMain(args []string, c cli.Config) error {
 	registerHandlers()
 
 	addr := net.JoinHostPort(conf.Address, strconv.Itoa(conf.Port))
-	log.Info("Now listening on ", addr)
-	return http.ListenAndServe(addr, nil)
+
+	tlsCA := conf.TLSCertFile
```

Comment:
I see no reason to create new names for these variables, and `tlsCA` is pretty misleading, given it's a certificate.